textdescriptives/subsetters.py

```diff
@@ -0,0 +1,36 @@
+"""Helpers to subset an extracted dataframe"""
```

Comment:
maybe put these in the same script as the Extractor?

Reply:
done in 1cf1527